redisinsight/api/src/modules/shared/services/instances-business/auto-discovery.service.ts

```diff
@@ -1,90 +0,0 @@
-import {
```

Comment:
Why did you remove this?

Reply:
My fault, i don't know when i did it. Restored it